### PR TITLE
Account Cards CRUD

### PR DESCRIPTION
**Pull request by nrprosper**
#### What does this PR do?
Have Rest endpoints on account cards

#### Description of Task to be completed?
Have the following endpoints working

POST `/api/cards`
GET `/api/cards`
GET `/api/cards/<card-id>`
DELETE `/api/cards/<card-id>`
PUT `/api/cards/<card-number>/change-pin`
PUT `/api/cards/<card-number>/pin-reset`

#### How should this be manually tested
After cloning you can use postman to check if this routes are working...
POST `/api/cards` it has no body for now. Everything is dynamically generated on the backend (sensitive informaion like account card number, cvv and pin)

PUT `/api/cards/<card-number>/change-pin`
`Content-Type: application/json
{
  "oldPin": "",
  "newPin": ""
}`

PUT `/api/cards/<card-number>/pin-reset`
`Content-Type: application/json
{
  "newPin": "1234"
}`

### CODE REVIEW

**Review by RedJanvier — APPROVED** [review]
Please add a way to get a single card

**Review by RedJanvier — CHANGES_REQUESTED** [review]
1. Set a way to get a single card either by id or card number and
2. Also on create, the pin should be a randomly generated one instead of a similar one.
3. Also add a way to recover a card if someone forgot the old pin

File: .env [hunk 1]
```diff
@@ -0,0 +1,5 @@
+DB_URL=jdbc:mysql://db:3306/bms_db
```
Comment (RedJanvier) [inline .env]: This file should not be pushed! make sure to remove it from this PR.

**Review by RedJanvier — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (nrprosper) [reply .env]: .env fle is removed

**Review by RedJanvier — APPROVED** [review]
LGTM!🚀

**Pull request merged**